presentational/Translation.js

```diff
@@ -1,30 +1,64 @@
 import styled from 'styled-components'
+import TitleBar from './fragments/TitleBar'
 
-const TitleBar = styled.div``
 const Translations = styled.div`
   display: grid;
-  grid-template-columns: repeat(4, 1fr);
+  grid-template-columns: 90px 770px;
+  margin: 0 0 20px 0;
 `
-const WordClass = styled.h2`
+const WordClass = styled.span`
+  font-family: ${({ theme }) => theme.fonts.title};
+  font-weight: 600;
+  font-size: 22px;
   color: ${({ theme }) => theme.colors.primaryMuted};
+  text-align: right;
+  padding: 0 10px 17px;
```

Comment:
17 is a quite strange number. By opening this example in the browser it looks to me like this is to compensate for the 15px bottom margin in TranslationController. If you add `grid-row-gap: 15px;` to Translations you can skip both the padding here and the margin in TranslationContainer.

Then you have one place where you decide the row spacing. Otherwise, you have to always synchronize 2 places.

Reply:
oh hey that's very cool! thanks!